bad.rb

```diff
@@ -0,0 +1,7 @@
+class Bad
+  def asdf ds,dfdsf
+    "As"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.